src/redisearch_rs/trie_bencher/src/ffi.rs

```diff
@@ -1,7 +1,149 @@
-#![allow(non_upper_case_globals)]
-#![allow(non_camel_case_types)]
-#![allow(non_snake_case)]
-#![allow(improper_ctypes)]
-#![allow(dead_code)]
-#![allow(unsafe_op_in_unsafe_fn)]
-include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+//! This module provides FFI bindings for the C trie functions encapsulated in the [CTrieMap] implementation.
```

Comment:
Let's keep the `ffi` module scoped to bindings, with `CTrieMap`/other utils in their own modules as they used to be.